style.css

```diff
@@ -0,0 +1,74 @@
+html {
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #f4f4f4;
+  padding: 0;
+}
+
+.wrapper {
+  background-image: url(./Images/illustration-header-mobile.png);
+  background-color: #1c1a19;
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  color: white;
+  background-position-y: 22rem;
+  max-height: 90%;
+  width: 375px;
+  display: flex;
+  flex-direction: column;
+  margin: 5vh auto 5vh auto;
+}
+
+.wrapper > * {
+  flex: 1 1 auto;
+}
+
+.navbar-section {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  padding: 20px 24px;
+  background: #3c3a39;
+}
+
+header {
```

Comment:
[OPTIONAL] I would recommend using classes for styling instead of elements, this way your CSS code is easier to reuse and it will make it easier to design the desktop version.